rs/registry/canister/canister/canister.rs

```diff
@@ -398,6 +425,48 @@ fn get_chunk_(request: GetChunkRequest) -> Result<Chunk, String> {
     registry().get_chunk(request)
 }
 
+/// Modifies records with keys of the form "daniel_wong_{}".
+///
+/// Returns new version number.
+///
+/// Caller must be GOVERNANCE_CANISTER_ID.
+///
+/// Used in integration test(s) for large records.
+///
+/// At the same time, it is not harmful for this to be in release builds,
+/// because
+///
+///     1. Only Governance canister is allowed to call this.
+///     2. This does not appear in registry.did.
+///
+/// There are a couple of pieces of functionality here that cannot otherwise
+/// easily be accomplished:
+///
+///     1. Produce large record(s).
+///     2. Chunking is ALWAYS enabled.
+///
+/// Again, since this can only be called by the Governance canister, production
+/// is protected from prematurely (i.e. before clients are ready) doing those
+/// things.
+#[export_name = "canister_update mutate_daniel_wong"]
+fn mutate_daniel_wong() {
```

Comment:
We shouldn't have test only methods in production canisters.  It should at minimum be behind feature=test.

Also, please fix the name.

Reply:
Per another feedback (from Jason), I changed the name to mutate_daniel_wong_for_test. Not sure if that is what you are looking for, but optimistically resolving this. Feel free to re-open if not.

Reply:
It seems like a) we shouldn't be putting our names into the main part of the codebase, and b) we should give names that indicate intentions or behavior in a useful way.

It could be called mutate_{literally_anything_at_all} in the current scheme, so long as it doesn't overlap with real registry records, and still have the same justification for naming.

Perhaps mutating Daniel Wong is a very desirable feature, but it seems like something that the Registry would not normally be responsible for, and in fact, the only reason it's there is to test the high capacity types.  So maybe "mutate_test_high_capacity_records" or something like that would be a bit clearer.

Also, I'm wondering if you considered Jason's suggestion for testing at a different level of abstraction.

Reply:
> Please, see that thread.
Github makes it very hard to find specific threads once they're resolved but I will go look.